thunder/tests/test_core.py

```diff
@@ -2288,7 +2292,7 @@ def func(x, y, device):
     assert len(flatten_cse_trace.bound_symbols) == len(flatten_dce_trace.bound_symbols) - 3
     assert len([bsym for bsym in flatten_cse_trace.bound_symbols if bsym.sym.id == prims.PrimIDs.UNIFORM]) == 4
 
-    assert [t.name for t in tree_flatten(flatten_cse_trace.output)[0]] == ["t4", "t4", "t6", "t7", "t8", "t9", "t10"]
+    assert [t.name for t in tree_flatten(flatten_cse_trace.output)[0]] == ["t4", "t4", "t6", "t14", "t15", "t16", "t17"]
```

Comment:
For some reason Thunder JIT doesn't rename t14 to t7, t15 to t8, t16 to t9, t17 to t10. https://github.com/Lightning-AI/lightning-thunder/issues/946